src/views/Profile/PersonalInfo/RegionCityHandler.tsx

```diff
@@ -122,7 +122,11 @@ const RegionCityHandler = ({
           disablePortal
           options={selectRegions || regions}
           fullWidth
-          value={regionForValue}
+          value={
+            newAuthorValues.regionId
+              ? regions.find((region) => region.id === newAuthorValues.regionId)
```

Comment:
why we need `regionForValue` when selection does not match any region or is empty?

Reply:
maybe we don't need it at all. I removed it and used `newAuthorValues` to store the value.